src/components/Posts/EditPostModal.tsx

```diff
@@ -66,7 +66,7 @@ const EditPostModal = ({
   open,
   post,
 }: EditPostModalProps) => {
-  const user = useAppSelector((state) => state.user);
+  const username = useAppSelector((state) => state.user.username);
```

Comment:
I like that you decided it wasn't worth creating a selector here, since we only use it once.